### PR TITLE
fix(taskDetails): fixed the tooltips and the alignment of upload docu…

### PR DESCRIPTION
**Pull request by printf-twinkle**
fix(taskDetailsPage)
### Functionality:
Task details page bug fixes

### Solution:
Fixed the tooltips and the alignment of upload document icon

### Risk level:
- [ ] high 
- [ ] medium
- [x] low

### How to test:
Go to a task and view the task details page

### CODE REVIEW

File: src/app/body/task-details/upload-files/upload-files.component.html [hunk 1]
```diff
@@ -17,7 +17,7 @@ <h6 class="col-md-11 col-10 pt-2 mb-0">Upload Files</h6>
         <div class="col-md-1 col-2 pt-1">
             <form>
                 <div class="form-group">
-                    <label class="btn" for="exampleFormControlFile1"><span class="material-icons uploadIcon">cloud_upload</span></label>
+                    <span class="material-icons uploadIcon">cloud_upload</span>
```
Comment (viv9k) [inline src/app/body/task-details/upload-files/upload-files.component.html:20]: Better use mat-icon wala code. Check in header component for reference

File: src/app/body/task-details/upload-files/upload-files.component.html [hunk 2]
```diff
@@ -36,7 +36,7 @@ <h6 class="col-md-11 col-10 pt-2 mb-0">Upload Files</h6>
     <form>
         <div class="form-group">
             <label class="btn btn-outline-secondary btn-lg btn-block p-2" for="exampleFormControlFile1">
-              <h5><span class="material-icons">cloud_upload</span> Click to Upload Document</h5>
+              <h5><span class="material-icons" style="position: relative; top: 7px; right: 2px;">cloud_upload</span> Click to Upload Document</h5>
```
Comment (viv9k) [inline src/app/body/task-details/upload-files/upload-files.component.html:39]: The properties of icon is static, fix is to bootstrap standards. Use mat-icon and padding and a tag to fix this.

Comment (viv9k) [reply src/app/body/task-details/upload-files/upload-files.component.html:39]: find a better approach

**Review by viv9k — CHANGES_REQUESTED** [review]
Please do the required changes

File: src/app/body/task-details/upload-files/upload-files.component.html (hunk 1 above)
Comment (printf-twinkle) [reply src/app/body/task-details/upload-files/upload-files.component.html:20]: done

File: src/app/body/task-details/upload-files/upload-files.component.html (hunk 2 above)
Comment (printf-twinkle) [reply src/app/body/task-details/upload-files/upload-files.component.html:39]: done

**Review by viv9k — APPROVED** [review]
Approved

**Pull request merged**